docs/DeveloperGuide.md

```diff
@@ -269,6 +269,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: UC01 - Adding a student**
+
+**MSS**
+
+1. User enters a command to add a student with student details.
+2. Reeve saves student data into the students list and displays a success message.
+<br> Use case ends.
+
+**Extensions**
+
+* 1a. User provides input with missing compulsory fields
```

Comment:
might want to add a period here to standardize

Reply:
Right! Thanks for pointing it out! I have made the necessary changes.